### PR TITLE
Fix nil document_type error, refactor Hub::DocumentsController#create, and remove intake setting on hub uploaded documents

### PR DESCRIPTION
**Pull request by bytheway875**

### CODE REVIEW

File: app/controllers/hub/documents_controller.rb [hunk 1]
```diff
@@ -25,17 +25,8 @@ def new; end
     def edit; end
 
     def create
-      document_params[:upload].each do |file_upload|
-        Document.create!(
-          uploaded_by: current_user,
-          client: @client,
-          intake: @client.intake,
-          document_type: document_params[:document_type],
```
Comment (bytheway875) [inline app/controllers/hub/documents_controller.rb:33]: When we set the params directly like this, when document_params[:document_type] is nil, it explicitly sets document_type to nil. When a nil value is provided, the default value from the database cannot be applied and the transaction fails the field required validation. By refactoring to apply only the params that are passed, we can ensure that a nil value does not get set.

**Review by sniemeyer13 — APPROVED** [review]
👍🏻

**Review by bengolder — COMMENTED** [review]
Looks great! Thanks for the fix!

**Comment by bengolder** [comment]
Oops, too late on the review

**Pull request merged**